islandora_pathauto.module

```diff
@@ -319,3 +322,95 @@ class IslandoraFedoraObjectReadOnly implements ArrayAccess {
    */
   public function offsetUnset($offset) {}
 }
+
+/*
+ * Implements hook_form_alter()
+ * 
+ * Adds a TextField to Islandora Properties so the User can Change
+ * the URL Alias.
+ */
+function islandora_pathauto_form_alter(&$form , &$form_state, $form_id){
+    if( $form_id == 'islandora_object_properties_form' ){
```

Comment:
Control statements should have one space between the control keyword and opening parenthesis.
https://www.drupal.org/coding-standards#controlstruct
(occurs multiple times in this function)